docs/modules/user-guide/partials/proc_defining-attributes.adoc

```diff
@@ -1,14 +1,17 @@
-[id="proc_adding-attributes-to-a-devfile_{context}"]
-= Adding attributes to a devfile
+[id="proc_defining-attributes_{context}"]
+= Defining attributes
 
 [role="_abstract"]
-Use devfile attributes to configure various features and properties according to user and tooling needs. Attributes are implementation-dependent and written in free-form YAML. The following devfile objects can have attributes:
+Use devfile attributes to configure various features and properties according to user and tooling needs. Attributes are implementation-dependent and written in free-form YAML. 
```

Comment:
Would it help to name the user role in this sentence? For example, "As a developer, you can use..."

Reply:
Done